### DISCUSSION TITLE
Can I  synchronize  Github Copilot on another computer?

### DISCUSSION THREAD

**Discussion by trevoryang1129** [community/community#19903] [Copilot Conversations]
Hello guys:
           I use Github Copilot  for a long time in my work computer,it do me lots of help,make my work very efficient.but I use the same account login my home computer, the suggestion is less help.May I sync my work computer's traning data to my home computer ,so that I can get the same suggestions as my work computer

**Comment by huabin** [comment]
Hello @trevoryang1129

GitHub Copilot is not tied to a computer and it works on all your computers. The things you need to do are following.

1. Install an IDE that supports GitHub Copilot on another computer, e.g. VSCode

2. Install GitHub Copilot extension for that IDE.

3. Log into your GitHub account that uses GitHub Copilot as it requires.

It will works! Hope this can help you.

**Reply by trevoryang1129** [reply]
Hello Huabin:
that's what I do now.but obviously in my work computer, Copilot give me more useful suggestion,but in my home computer i got less suggestion.

**Reply by huabin** [reply]
I don't think it has anything to do with computers. Copilot is an AI-based tool and there are many factors that affect it. It is normal that Copilot gives different suggestions, even in the same project on the same computer.

**Reply by inebhedj** [reply]
Not possible. Github Copilot does not synchronize individual instances even under the same account (see chat history).
